ouroboros-consensus-shelley-test/src/Test/Consensus/Shelley/Examples.hs

```diff
@@ -78,6 +78,7 @@ fromShelleyLedgerExamples ShelleyLedgerExamples {
     , exampleQuery            = queries
     , exampleResult           = results
     , exampleAnnTip           = unlabelled annTip
+    , exampleLedgerConfig     = mempty
```

Comment:
See similar comment in the Cardano `Examples` module.

Reply:
I can easily stick a `ShelleyLedgerConfig era` in here, but we'd end up with the same example for each era. Perhaps I can use something in the `ShelleyLedgerExamples` parameter as a source of noise. Did you have something else in mind?

Reply:
This will be replaced with some dummy example

Reply:
Fixed! Marking Resolved.